src/chainparams.cpp

```diff
@@ -205,10 +205,10 @@ class CTestNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_SEGWIT].nTimeout = 1493596800; // May 1st 2017
 
         // The best chain should have at least this much work.
-        consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000000000001f057509eba81aed91");
+        consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000000000002830dab7f76dbb7d63");
```

Comment:
Another possibility potentially less confusing for review it's also going back 20k blocks for the nMinimumChainWork to match the chain work of the chosen defaultAssumeValid.
As explained to me [last time](https://github.com/bitcoin/bitcoin/pull/9779#issuecomment-280536291) they don't need to match, but perhaps someone different will ask the same question on each major release and the gain of them not matching is dubious to me (it's also more work for the coder if they don't match).